package/python-kiwi_keg.spec

```diff
@@ -0,0 +1,138 @@
+#
+# spec file for package python-kiwi_keg
+#
+# Copyright (c) 2022 SUSE Software Solutions Germany GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+#
+# Please submit bugfixes or comments via:
+#
+#       https://github.com/SUSE-Enceladus/keg/issues
+#
+
+# If they aren't provided by a system installed macro, define them
+%{!?_defaultdocdir: %global _defaultdocdir %{_datadir}/doc}
+%{!?__python3: %global __python3 /usr/bin/python3}
+
+%if %{undefined python3_sitelib}
+%global python3_sitelib %(%{__python3} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
+%endif
+
+%if 0%{?el7}
+%global python3_pkgversion 36
+%else
+%{!?python3_pkgversion:%global python3_pkgversion 3}
+%endif
+
+%if 0%{?debian} || 0%{?ubuntu}
+%global is_deb 1
+%global pygroup python
+%global sysgroup admin
+%global develsuffix dev
+%else
+%global pygroup Development/Languages/Python
+%global sysgroup System/Management
+%global develsuffix devel
+%endif
+
+Name:           python-kiwi_keg
```

Comment:
Should be `python3-kiwi-keg` since we are not building a Python 2 version. While the package provides an executable and an API and it would be appropriately named `python-kiwi-keg` the precedence we have set is that we use `python3-` as prefix when no Python2 version is being built. Also means the file name of the spec file needs to change.

Reply:
Almost no package in Factory uses `python3` as prefix for the package name. `python` appears to be the standard (including python packages that have python2 build disabled). The binary packages are automatically produced with the appropriate prefix, i.e. `python38`, `python39`, and `python310` in Factory and `python3` in SLE/Leap.